### PR TITLE
Upgrade to eventing v1 apis and eventing 0.22

### PR DESCRIPTION
**Pull request by grantr**
_Alternate, smaller-scope version of #2197, upgrading to 0.22 instead of head (0.23)._

The v1beta1 versions of Broker and Trigger have been removed from upstream. The v1 version has an identical schema, so replacing is just import juggling.

This should unblock the auto dependency update.

## Proposed Changes
* 🗑️ Remove support for Broker and Trigger v1beta1

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -169,6 +169,8 @@ func TestChannelChain(t *testing.T) {
 	e2ehelpers.ChannelChainTestHelper(
 		context.Background(),
 		t,
+		// TODO(tommyreddad): eventing 0.22 e2e/helpers has a bug where SubscriptionV1beta1 and SubscriptionV1 are flipped in the ChannelChainTestHelper.
```
Comment (grantr) [inline test/e2e/e2e_test.go:172]: Create an issue to track this

Comment (tommyreddad) [reply test/e2e/e2e_test.go:172]: Done. https://github.com/google/knative-gcp/issues/2202

**Pull request merged**